cdc/owner.go

```diff
@@ -294,6 +294,47 @@ func (o *Owner) newChangeFeed(
 	return cf, nil
 }
 
+func (o *Owner) checkAndCleanTasksInfo(ctx context.Context) error {
```

Comment:
This is a compatibility hack between v4.0.0 and v4.0.1, leave a comment about this, as it is not a general resource clean API